src/asm_syntax.hpp

```diff
@@ -11,24 +11,28 @@
 #include "crab/variable.hpp"
 #include "spec_type_descriptors.hpp"
 
+constexpr char STACK_FRAME_DELIMITER = '/';
+
 namespace crab {
 struct label_t {
```

Comment:
My personal taste: rename label to something else ("_label_t" maybe) and add a new type named "label_t" that holds the label and the context.
Also I'd prefer the context to have an easier to handle type (`vector<_label_t>`).